ArcTouchTMDb/ArcTouchTMDb.Core/Models/Movie.cs

```diff
@@ -0,0 +1,26 @@
+﻿using System;
+using System.Collections.Generic;
+
+namespace ArcTouchTMDb.Core
+{
+	public class Movie
+	{
+		public string poster_path { get; set; }
```

Comment:
Also, Json.Net is not case-sensitive when mapping, so a property with the name `Overview` would map correctly to Json field `overview` without the need of any attribute.